src/mbgl/map/map.cpp

```diff
@@ -421,11 +421,24 @@ CameraOptions Map::cameraForLatLngs(const std::vector<LatLng>& latLngs, const Ed
     // CameraOptions origin is at the top-left corner.
     centerPixel.y = viewportHeight - centerPixel.y;
 
-    options.center = latLngForPixel(centerPixel);
+    options.center = transform.screenCoordinateToLatLng(centerPixel);
     options.zoom = zoom;
     return options;
 }
 
+CameraOptions Map::cameraForLatLngs(const std::vector<LatLng>& latLngs, const EdgeInsets& padding) const {
+    return mbgl::cameraForLatLngs(latLngs, impl->transform, padding);
+}
+
+CameraOptions Map::cameraForLatLngs(const std::vector<LatLng>& latLngs, double heading, const EdgeInsets& padding) const {
+    double angle = -heading * util::DEG2RAD;
```

Comment:
The convention throughout mbgl seems to be to use the term "angle" for what might be called "heading" or "bearing" elsewhere.